### ISSUE TITLE
animate_topomap() without drawing contours?

### ISSUE THREAD

**Issue by PappaBu** [mne-tools/mne-python#7938]
Is it possible to create an animation using [`animate_topomap()`](https://mne.tools/stable/generated/mne.Evoked.html#mne.Evoked.animate_topomap) without drawing contours?
I can hide contours in [`plot_topomap()`](https://mne.tools/stable/generated/mne.Evoked.html#mne.Evoked.plot_topomap) with option `contours=0`, but I can not find something like this in the `animate_topomap()`.
Is there a way around this limitation?

**Comment by agramfort** [comment]
yes this option is not implemented. Pull request very welcome to add this.

>

**Comment by PappaBu** [comment]
Sorry, but I'm new and just an ordinary user. How can I ask to add this option to the MNE-Python?

**Comment by drammock** [comment]
@PappaBu this issue will serve as a feature request.  No guarantees on how soon it will get added though.
